app/src/main/java/com/github/se/travelpouch/MainActivity.kt

```diff
@@ -42,12 +44,29 @@ fun TravelPouchApp() {
   val navigationActions = NavigationActions(navController)
   val listTravelViewModel: ListTravelViewModel = viewModel(factory = ListTravelViewModel.Factory)
 
-  NavHost(navController = navController, startDestination = Route.OVERVIEW) {
+  NavHost(navController = navController, startDestination = Route.AUTH) {
     navigation(
-        startDestination = Screen.ADD_TRAVEL,
-        route = Route.OVERVIEW,
+        startDestination = Screen.AUTH,
+        route = Route.AUTH,
     ) {
+      composable(Screen.AUTH) { SignInScreen(navigationActions) }
+    }
+
+    navigation(
+        startDestination = Screen.GREETING,
+        route = Route.GREETING,
+    ) {
+      composable(Screen.GREETING) { Greeting() }
       composable(Screen.ADD_TRAVEL) { AddTravelScreen(listTravelViewModel, navigationActions) }
     }
   }
 }
+
+@Composable
+fun Greeting() {
```

Comment:
Linked to my other comment, this could be removed

Reply:
See my answer, do you agree to keep it temporarily ?